src/seedu/addressbook/AddressBook.java

```diff
@@ -221,7 +221,7 @@ private static void showWelcomeMessage() {
         showToUser(DIVIDER, DIVIDER, VERSION, MESSAGE_WELCOME, DIVIDER);
     }
 
-    private static void showResultToUser(String result) {
+    private static void showResultToUserWithDivider(String result) {
```

Comment:
May want to have some comments to explain what is this method doing now as the term "divider" may not be clear to others.